ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
Maybe I am misusing the term component or confused.  What I am
requesting this the rationale for storing "obfuscated ids of
other objects in the compositeId".  This is confusing, given the
goal of obfuscation is to keep the full id string from being
over-interpreted by client.

Danny Colligan <notifications@github.com> writes:
> In ga4gh/datamodel/__init__.py:
> 
> > @@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
> >      """
> >      fields = DatasetCompoundId.fields + ['variantSet']
> >      containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
> > +    obfuscators = [DatasetCompoundId, None]
> 
> We are not obfuscating each component. We are storing the obfuscated ids of
> other objects in the compositeId -- those other objects are, specifically, the
> parent objects of the object to which the compositeId corresponds.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>